appveyor.yml

```diff
@@ -8,40 +8,25 @@ environment:
   - GOBIN: c:\go
     APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
-services:
-- mysql
-- postgresql
-
-install:
-- cmd: >-
-    choco install mingw
-
-    choco install dep
-
-    set PATH=%GOPATH%\bin;%GOBIN%\bin;c:\tools\mingw64\bin;C:\ProgramData\chocolatey\lib\dep\tools;%PATH%
-
-    echo %PATH%
-
-    echo %GOPATH%
-
-    go version
-
-    go env
 build:
   verbosity: normal
+
 deploy: false
 
-before_build:
+stack: go 1.11
+
+install:
 - cmd: >-
-    go get -v -u github.com/mattn/go-sqlite3
+    choco install mingw
 
-    go get -v -u -t ./...
+    set PATH=%GOPATH%\bin;%GOBIN%\bin;c:\tools\mingw64\bin;%PATH%
 
 build_script:
 - cmd: >-
-    go build -v -tags sqlite github.com/gobuffalo/buffalo/buffalo
+    set GO111MODULE=on
 
-    go install -tags sqlite github.com/gobuffalo/buffalo/buffalo
+    go build -v github.com/gobuffalo/buffalo/buffalo
 
 test_script:
-- cmd: for /f "" %%G in ('go list github.com/gobuffalo/buffalo/... ^| find /i /v "/vendor/"') do ( go test -tags "sqlite integration_test" %%G & IF ERRORLEVEL == 1 EXIT 1)
+  - set GO111MODULE=on
+  - go test -race ./...
```

Comment:
Can we keep the integration tests? That's one of the reasons we have a specific build for Windows.